recipes/kernax/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: kernax
+  version: 0.0.1
+
+source:
+  # git_url: https://gitlab.com/drti/kernax.git
+  url: https://gitlab.com/drti/kernax/-/archive/main/kernax-main.tar.gz
```

Comment:
I'm concerned that the url isn't really a release tied to the `version`. I'm not familiar with how gitlab does releases, but is there a less generic tarball available that clearly is tied to one of the tagged releases?

Reply:
I found where the tagget tarballs are located on gitlab. Thank you for spotting that.